addons/godot-firebase/auth/auth.gd

```diff
@@ -320,6 +332,7 @@ func _on_FirebaseAuth_request_completed(result : int, response_code : int, heade
     var json_result = JSON.parse(bod)
     if json_result.error != OK:
         Firebase._printerr("Error while parsing body json")
+        emit_signal("auth_request", "Error while parsing body json", auth)
```

Comment:
Let's mention that it's the auth body json here.

Reply:
Want that instead of the auth object itself? so: `emit_signal("auth_request", "Error while parsing body json", json_result)` or `emit_signal("auth_request", "Error while parsing body json", json_result.error)`?

Reply:
See updated, though now realizing I might have misunderstood what you meant (str change vs var passed into signal change), but let me know what you think.